conviva/docs/conviva_metrics.md

```diff
@@ -0,0 +1,47 @@
+## Conviva Monitor Metric Parameters and Metrics
+
+`Metric Parameters` are conviva monitor `metricParameter` configuration values. `Metrics` are the metrics that get reported to SignalFx
```

Comment:
I'm not sure that the words Metric Parameters or Metrics should be inside back-ticks.